tasks/main.yml

```diff
@@ -35,4 +35,13 @@
   ansible.builtin.import_tasks: configure.yml
   become: yes
 
+- name: "Force all notified handlers to run at this point. Required for feature flags."
+  ansible.builtin.meta: "flush_handlers"
```

Comment:
We should be careful with flushing handlers. Did you check whether this has an impact on subsequent roles?